packages/related-content/style.module.css

```diff
@@ -0,0 +1,70 @@
+.wrapper {
+  composes: g-grid from global;
+  composes: g-grid-container from global;
+  padding-top: 64px;
+  padding-bottom: 64px;
+
+  @media (--large) {
+    padding-top: 88px;
+    padding-bottom: 88px;
+  }
+  &.dark {
+    background-color: var(--wpl-neutral-900);
+  }
+}
+
+.textStack {
+  grid-column: 1 / -1;
+  margin-bottom: 24px;
+
+  @media (--medium) {
+    grid-column: 1 / span 7;
+    margin-bottom: 32px;
+  }
+
+  @media (--large) {
+    grid-column: 1 / span 6;
+    margin-bottom: 32px;
+  }
+}
+
+.headline {
+  composes: g-type-display-3 from global;
+  color: var(--wpl-neutral-900);
+  margin: 0 0 16px 0;
+
+  &.dark {
+    color: white;
+  }
+}
+
+.description {
+  composes: g-type-body from global;
+  color: var(--wpl-neutral-500);
+  margin: 0;
+
+  &.dark {
+    color: white;
+  }
+}
+
+.cards {
+  --columns: 1;
+
+  grid-column: 1 / -1;
+  display: grid;
+  grid-gap: 32px;
+  grid-template-columns: repeat(var(--columns), 1fr);
+  margin-bottom: 32px;
+
+  @media (--medium) {
+    --columns: 2;
+  }
+  @media (--large) {
+    --columns: 4;
```

Comment:
Just to confirm expectations, if there are three cards provided, we still render 4 columns across vs 3 columns across?

Reply:
<img width="1720" alt="CleanShot 2022-10-03 at 10 42 25@2x" src="https://user-images.githubusercontent.com/825855/193605919-8ce3a766-3a2c-4e09-bc33-55679d4458f2.png">

Bottom example rendering 3 columns vs 4.

Reply:
> Just to confirm expectations, if there are three cards provided, we still render 4 columns across vs 3 columns across?

My understanding is that this component can only render 4 cards.

Reply:
Interesting, so to make use of RelatedContent, folks will need to provide at least 4 cards. Or it's ok that less than 4 cards are provided, they just will still render within 4 columns? I could see the need to use less based on some of content the content I have seen on .io sites.

If we've decided that is a requirement, what do you think about adding a length check to the cards array that at least 4 cards are provided then or update types to reflect 4 cards?

```tsx
cards: [RelatedContentCardProps, RelatedContentCardProps, RelatedContentCardProps, RelatedContentCardProps]
```

Reply:
FWIW also seeing less than and more than 4 related resources being used in the work @EnMod is doing with SI Partners.

- https://hashicorp-www-next-git-nqsi-update-template-hashicorp.vercel.app/partners/systems-integrators/adfinis
- https://hashicorp-www-next-git-nqsi-update-template-hashicorp.vercel.app/partners/systems-integrators/amazic

Reply:
>> Interesting, so to make use of RelatedContent, folks will need to provide at least 4 cards. Or it's ok that less than 4 cards are provided, they just will still render within 4 columns? I could see the need to use less based on some of content the content I have seen on .io sites.

FWIW - I think this component should allow for multiple cards and not be limited to only 4. I wonder if it makes sense to enable that and make adjustments later. Especially if we're already seeing deviations from the initial requirements.

Reply:
@nandereck I agree: I think it makes sense to go with how it's currently implemented (render with 4 columns regardless of count, do not limit to 4 cards minimum). Especially with deviations already existing. 

As an aside, this implementation is aligned with how local related resources components are currently implemented, an example being on [our thank you page](https://www.hashicorp.com/thank-you) :+1: